skyportal/tests/api/test_spectrum.py

```diff
@@ -5,7 +5,7 @@
 
 def test_token_user_post_get_spectrum_data(upload_data_token, public_source):
     status, data = api('POST', 'spectrum',
-                       data={'source_id': str(public_source.id),
+                       data={'obj_id': str(public_source.id),
```

Comment:
`public_obj`?

Reply:
`public_source` implies it's associated with a `Source` entry -- this feels clearer to me